### PR TITLE
strip trailing whitespace from lookup types

### PR DESCRIPTION
**Pull request by hfaulds**
We're experiencing an issue in one mls where they have trailing whitespace on these values.

We already strip trailing whitespace from values in our codebase and then we fail to resolve them to long values.

This could be put behind a bit of config but I'm struggling to see any scenario where this would be bad idea.

### CODE REVIEW

**Comment by phiggins** [comment]
LGTM :THUMBSUP:, but CI is not happy.

**Comment by hfaulds** [comment]
Hmm, travis appears to have been complaining that our `gemspec` and `Gemfile` mocha versions differed. I updated the `gemspec` and `Rakefile` versions to match the `Gemfile` but I'm not sure of the greater implications of this or why I had to make this change.

**Comment by hfaulds** [comment]
Nevermind, mocha had already beed updated, I just branched off of an old version of master when I made this PR

**Comment by hfaulds** [comment]
I need to investigate how this will affect other mlses we're using before we merge this.

File: lib/rets/metadata/lookup_type.rb [hunk 1]
```diff
@@ -4,8 +4,8 @@ class LookupType
       attr_reader :long_value, :value
 
       def initialize(lookup_type_fragment)
-        @value       = lookup_type_fragment["Value"]
-        @long_value  = lookup_type_fragment["LongValue"]
+        @value       = lookup_type_fragment["Value"].rstrip
+        @long_value  = lookup_type_fragment["LongValue"].rstrip
```
Comment (dougcole) [inline lib/rets/metadata/lookup_type.rb]: every other `strip` in this codebase is just a standard `strip`, not `rstrip`. What's the reason to switch here?

Comment (hfaulds) [inline lib/rets/metadata/lookup_type.rb]: This came from a mis-understanding on my part about some code we have in `estately`'s private repos. We use `rstrip` on values but this happens after they're resolved anyway. I'll switch it to `strip`

Comment (hfaulds) [inline lib/rets/metadata/lookup_type.rb]: I double-checked with a script and `strip` doesn't affect any types that aren't affected by `rstrip` so it seems safe to use either.

**Comment by hfaulds** [comment]
This affects ~10 of the mlses we integrate with at varying levels of severity. For some lookup tables it affects 1 lookup type such that it never resolves properly whereas others it affects far more.

I investigate a few places where this was affecting us and could only find instances of this PR fixing things.

When using `COMPACT-DECODED` this change won't affect anything.

**Comment by dougcole** [comment]
Thanks @hfaulds!

**Pull request merged**